worlds/rabi_ribi/docs/en_Rabi-Ribi.md

```diff
@@ -0,0 +1,28 @@
+# Rabi-Ribi
+
+## Where is the options page?
+
+The [player options page for this game](../player-options) contains all the options you need to configure and export a
+config file.
+
+## What does randomization do to this game?
+
+Items which the player would normally acquire throughout the game have been moved around. Logic remains, so the game is
+always able to be completed, but because of the item shuffle, the player may need to access certain areas before they
+would be expected to in the vanilla game.
+
+## What is the goal of Rabi-Ribi in Archipelago
+
+The goal of Rabi-Ribi is to find 5 Easter Egg items shuffled throughout Rabi Rabi Island. Collecting the fifth Easter Egg
+will complete your seed.
```

Comment:
Suggest making a quick note about how the game can normally be beaten with 0% items, so beating the final boss isn't a very interesting goal (although 5 eggs + boss might be)

Reply:
Agreed. I think people who have played the game would be aware that the game is beatable in 0%, but there's def people that do their first play of a game in a randomizer.